src/cluster/setup.rst

```diff
@@ -0,0 +1,160 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _cluster/setup:
+
+=====
+Setup
+=====
+
+Everything you need to know to prepare the cluster for the installation of
+CouchDB.
+
+Firewall
+========
+
+If you do not have a firewall between your servers, then you can skip this.
+
+CouchDB in cluster mode uses the port ``5984`` just as standalone, but is also
+uses ``5986`` for the admin interface.
+
+Erlang uses TCP port ``4369`` (EPMD) to find other nodes, so all servers must be
+able to speak to each other on this port. In an Erlang Cluster, all nodes are
+connected to all other nodes. A mesh.
+
+.. warning::
+    Do not open up this port, ``4369`` to the Internet, or any other
+    untrusted network. Failing to close this port will leave you wide open!
```

Comment:
Good recommendation. However, you're not very wide opened since cookie is required to make a connection.